openedx/core/djangoapps/xblock/rest_api/views.py

```diff
@@ -213,6 +215,20 @@ def xblock_handler(
     return response
 
 
+@api_view(['GET'])
+@view_auth_classes(is_authenticated=False)
+def get_block_olx_view(
+    request,
+    usage_key: UsageKeyV2,
+    version: LatestVersion | int = LatestVersion.AUTO,
+):
```

Comment:
Please add a docstring. `""" Get the OLX (XML serialization) of the specified XBlock """`

Reply:
Updated https://github.com/openedx/edx-platform/pull/35932/commits/46ac9e2da497011bba8558679d890164c8948fb7